### PR TITLE
Allow batched payments

### PR DESCRIPTION
**Pull request by ggruszczynski**

### CODE REVIEW

File: apps/rendering/resources/ImgVerificator.py [hunk 1]
```diff
@@ -119,10 +119,10 @@ def crop_img_relative(self, img, crop_window):
         p.load_from_pil_object(cropped_img, img.get_name())
         return p
 
-    def get_random_crop_window(self, coverage=0.2, window=(0, 1, 0, 1)):
+    def get_random_crop_window(self, coverage=0.33, window=(0, 1, 0, 1)):
```
Comment (badb) [inline apps/rendering/resources/ImgVerificator.py:122]: Please, try to avoid adding changes that are not connected to the PR topic.

Comment (ggruszczynski) [reply apps/rendering/resources/ImgVerificator.py:122]: try avoid means 'revert' or 'dont do this in the future' ?

Comment (badb) [reply apps/rendering/resources/ImgVerificator.py:122]: Don't do this in the future.

File: lint.sh [hunk 2]
```diff
@@ -34,6 +33,11 @@ GREEN=$(
 ) 2>/dev/null
 RESET=$(tput sgr0) 2>/dev/null
 
+if ! git diff-index --quiet HEAD --; then
```
Comment (badb) [inline lint.sh]: Put please revert this. Put this in separate PR if you really need this one (but  please let us know why).

**Review by badb — APPROVED** [review]
It can be merged after this linter changes are reverted or very well argumented.

File: lint.sh (hunk 2 above)
Comment (ggruszczynski) [reply lint.sh]: reverted
